<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="osc_tabs">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>781</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="osc1_tab">
     <attribute name="title">
      <string>Osc1</string>
     </attribute>
     <widget class="QWidget" name="osc_group_1" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>761</width>
        <height>371</height>
       </rect>
      </property>
      <widget class="QRadioButton" name="noise_radio_1">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>50</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Noise</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_wave_osc1</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="saw_radio_1">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>50</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Saw</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_wave_osc1</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="x_radio_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>X-Axis</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_axis_osc1</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="sine_radio_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Sine</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_wave_osc1</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="square_radio_1">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>50</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Square</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_wave_osc1</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="z_radio_1">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>10</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Z-Axis</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_axis_osc1</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="tri_radio_1">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Triangular</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_wave_osc1</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="y_radio_1">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Y-Axis</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_axis_osc1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>741</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="osc_plot_1"/>
      </widget>
      <widget class="QSlider" name="downsample_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>291</width>
         <height>18</height>
        </rect>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="stretch_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>330</y>
         <width>291</width>
         <height>18</height>
        </rect>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>260</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Downsampling</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>310</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Strech</string>
       </property>
      </widget>
     </widget>
     <widget class="QCheckBox" name="osc_enabled_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Enabled</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>761</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="osc2_tab">
     <attribute name="title">
      <string>Osc2</string>
     </attribute>
     <widget class="QWidget" name="osc_group_2" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>761</width>
        <height>371</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>741</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="osc_plot_2"/>
      </widget>
      <widget class="QRadioButton" name="saw_radio_2">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>50</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Saw</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_wave_osc2</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="z_radio_2">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>10</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Z-Axis</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_axis_osc2</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="x_radio_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>X-Axis</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_axis_osc2</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="tri_radio_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Triangular</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_wave_osc2</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="y_radio_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Y-Axis</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_axis_osc2</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="noise_radio_2">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>50</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Noise</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_wave_osc2</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="sine_radio_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Sine</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_wave_osc2</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="square_radio_2">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>50</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Square</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_wave_osc2</string>
       </attribute>
      </widget>
      <widget class="QSlider" name="stretch_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>330</y>
         <width>291</width>
         <height>18</height>
        </rect>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="downsample_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>291</width>
         <height>18</height>
        </rect>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>260</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Downsampling</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>310</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Strech</string>
       </property>
      </widget>
     </widget>
     <widget class="QCheckBox" name="osc_enabled_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Enabled</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>761</width>
        <height>31</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="osc3_tab">
     <attribute name="title">
      <string>Osc3</string>
     </attribute>
     <widget class="QWidget" name="osc_group_3" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>39</y>
        <width>761</width>
        <height>371</height>
       </rect>
      </property>
      <widget class="QRadioButton" name="z_radio_3">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>10</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Z-Axis</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_wave_osc3</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="square_radio_3">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>50</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Square</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_axis_osc3</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="x_radio_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>X-Axis</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_wave_osc3</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="tri_radio_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Triangular</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_axis_osc3</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="y_radio_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Y-Axis</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_wave_osc3</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="saw_radio_3">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>50</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Saw</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_axis_osc3</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>741</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="osc_plot_3"/>
      </widget>
      <widget class="QRadioButton" name="noise_radio_3">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>50</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Noise</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_axis_osc3</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="sine_radio_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>102</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Sine</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btn_group_axis_osc3</string>
       </attribute>
      </widget>
      <widget class="QSlider" name="stretch_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>330</y>
         <width>291</width>
         <height>18</height>
        </rect>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="downsample_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>291</width>
         <height>18</height>
        </rect>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>310</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Strech</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>260</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Downsampling</string>
       </property>
      </widget>
     </widget>
     <widget class="QCheckBox" name="osc_enabled_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Enabled</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>21</y>
        <width>771</width>
        <height>31</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="bt_address">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>B8:AE:6E:EF:D2:D0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_connect">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="record">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QLabel" name="connection_label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Not connected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>540</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>osc_enabled_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>osc_group_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>osc_enabled_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>osc_group_1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>osc_enabled_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>osc_group_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="btn_group_wave_osc1"/>
  <buttongroup name="btn_group_axis_osc3"/>
  <buttongroup name="btn_group_axis_osc2"/>
  <buttongroup name="btn_group_axis_osc1"/>
  <buttongroup name="btn_group_wave_osc2"/>
  <buttongroup name="btn_group_wave_osc3"/>
 </buttongroups>
</ui>
